discogs_alert/client.py

```diff
@@ -113,14 +119,22 @@ def __init__(self, user_agent: str, *args, **kwargs):
             "--incognito",
             f"--user-agent={self.user_agent.random}",  # initialize with random user-agent
         ]
+        if os.geteuid() == 0:
+            # running as root
+            options_arguments.append("--no-sandbox")
```

Comment:
@michaelhball like this, in docker it'll  run as root, and other (Windows) users won't be affected